src/components/resources/StakeTab.vue

```diff
@@ -101,23 +101,23 @@ export default defineComponent({
                 receiver: this.stakingAccount.toLowerCase(),
                 transfer: false,
                 stake_cpu_quantity:
-          parseFloat(this.cpuTokens) > 0
-              ? `${parseFloat(this.cpuTokens).toFixed(4)} ${symbol}`
-              : `0.0000 ${symbol}`,
+                    parseFloat(this.cpuTokens) > 0
+                        ? `${parseFloat(this.cpuTokens).toFixed(4)} ${symbol}`
+                        : `0.0000 ${symbol}`,
```

Comment:
In this case, we should not change the numbers because it is part of the parameters of a smart contract call which takes precision very strictly. So if we change it, will fail.